tty-features.c

```diff
@@ -71,6 +71,18 @@ static const struct tty_feature tty_feature_osc7 = {
 	0
 };
 
+/* Terminal has OSC 133 semantic prompt. */
+static const char *const tty_feature_osc133_capabilities[] = {
+	"Swd=\\E]7;",
+	"fsl=\\a",
```

Comment:
I don't think you need this (or any change to `tty-features.c`) because there is no point in forwarding these sequences to the terminal outside - as soon as you switch pane or window they will all be invalid. So it doesn't matter if the terminal outside tmux supports it.

Reply:
So if I revert all the changes in `tty-features.c`, it will stil keep working? If that's the case, I'll remove the changes from this file.

Other than that, I've tested the feature and as far as I can tell it's working alright. I can jump backward and forward to the prompts, and the cursor is placed at the first cell of the prompt line.

Reply:
Yes it should it keep working. I'll take a look at it after you've done that (probably tomorrow or Mon).